docs: add description about debugging hours (#17)
falcosecurity/office-hours: DEBUGGING_HOURS.md

# Debugging hours

Debugging hours play a role when someone wants the help of others debugging an issue
they have with Flaco, either if it is an issue using it or developing against it!

The other purpose of the debugging hours is to offload discussions from the office hours
to a meeting solely dedicated to a specific issue.

Debugging hours are not scheduled in advance, what happens is that the person that wants to propose a topic

> just opens an issue with `/kind debugging-hours` proposing a debugging topic and put your availability to do a call in it.

Reviewer: ```suggestion
just opens an issue with `/kind debugging-hours` proposing a debugging topic and puts in his/her availability to do a call.
```

> just opens an issue with `/kind debugging-hours` proposing a debugging topic and puts in his/her availability to do a call.

Author: I don’t like the gender pronouns here @leodido

Reviewer 2: I would like to add that `reproducible environment is ready?` question or we want to make it mandatory to avoid errands
